frontend/src/layout/lemonade/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -24,15 +24,24 @@
 .Breadcrumbs__breadcrumb {
     display: flex;
     align-items: center;
```

Comment:
Let's give `.Breadcrumbs__breadcrumb` `height: 2rem` to give popups a bit more breathing room like so:
![room](https://user-images.githubusercontent.com/4550621/141378975-4a559189-54b8-41d9-a3e0-3170e9c5005f.png)
Though will probably require slightly downsizing the `Breadcrumbs` and `page-title` `margin-top`s so that there isn't too much room there.